### ISSUE TITLE
Creation of segmentation dataset doesn't crash when wrong label encoding is given

### ISSUE THREAD

**Issue by klaimane** [NVIDIA/DIGITS#1284]
When the actual label encoding is png and the parameter in digits is "None" there is no error. The creation runs it's course and the resulting dataset has no label images in it.

This can cause time/resource waste if you let the task run for hours on a big dataset.

**Comment by gheinrich** [comment]
I don't think this is a bug...  it's up to the user to choose the best encoding.

**Comment by klaimane** [comment]
You're essentially right, but I think the DIGITS process "knows" that something is going wrong, since it copies feature images to the dataset directory but not label images. If it would throw an error, this would free up computing resources and show the user there is an issue with the job more quickly.
Not a bug but perhaps an enhancement request?

**Comment by gheinrich** [comment]
Thanks, it's a legitimate enhancement request. Keeping open.

**Comment by gheinrich** [comment]
https://github.com/NVIDIA/DIGITS/pull/1290 helps, to some extent.

**Comment by klaimane** [comment]
I think it's the most sensible solution a user could hope for. 
Thanks!
